lib/mix/lib/mix/tasks/new.ex

```diff
@@ -139,8 +146,7 @@ defmodule Mix.Tasks.New do
     Inside your project, you will find an apps/ directory
     where you can create and host many apps:
 
-        cd #{path}
-        cd apps
+        #{cd_path(path)}cd apps
```

Comment:
I would prefer to keep them as two separate steps because it is not really required to run the last two commands (cd apps and mix new).